client/my-sites/upgrades/domain-management/transfer/utils.js

```diff
@@ -0,0 +1,101 @@
+/**
```

Comment:
I renamed the file to `shared.js`, I didn't want to create a file to put a single function in it.